application_examples/google_Colaboratory/agentQL.ipynb

```diff
@@ -90,30 +62,46 @@
       },
       "outputs": [],
       "source": [
-        "import agentql\n",
+        "from agentql.ext.playwright.async_api import Page\n",
+        "from playwright.async_api import async_playwright\n",
         "\n",
         "\n",
-        "session = await agentql.start_async_session(\"https://www.google.com\")\n",
+        "URL = \"https://scrapeme.live/shop\"\n",
         "\n",
         "QUERY = \"\"\"\n",
         "{\n",
-        "    search_box\n",
-        "    search_btn\n",
-        "    about_link\n",
+        "    search_products_box\n",
```

Comment:
Sounds good! Just fyi, the reason I specify `search_products_box` instead of `search_box` is that there are two search boxes on the scrapeme.live/shop website. And the query need to specify `search_products_box` in order to find the desired search field.